### PR TITLE
Document --timeout in livemedia-creator man page

### PR DESCRIPTION
**Pull request by atodorov**

### CODE REVIEW

File: docs/livemedia-creator.1 [hunk 1]
```diff
@@ -184,6 +185,10 @@ volume id
 \fB\-\-squashfs_args SQUASHFS_ARGS\fR
 additional squashfs args
 
+.TP
+\fB\-\-timeout TIMEOUT\fR
+Cancel installer after X minutes
```
Comment (vpodzime) [inline docs/livemedia-creator.1]: *"Terminate the installation"* sounds better to me, but native speakers should probably tell us what is the right wording here.

**Comment by vpodzime** [comment]
Looks good to me otherwise.

File: docs/livemedia-creator.1 (hunk 1 above)
Comment (atodorov) [inline docs/livemedia-creator.1]: I've taken this from --help description. Will update both if wording changes.

**Comment by bcl** [comment]
merged, thanks.

**Pull request closed without merging**